mihaela-etropolska/Homework3/calculator-style.css

```diff
@@ -0,0 +1,50 @@
+h2 {
+	margin-left: 100px;
+}
+
+table {
+	-webkit-border-radius: 10px;
+	-moz-border-radius: 10px;
+	border-radius: 10px;
+	border: solid black;
+	width: 19em;
+	height: 21em;
+	background-color: blue;
+}
+
+th {
+	background-color:white;
+	height: 60px;
+	text-align: right;
+	font-size:40px;
+
+	-webkit-border-radius: 10px;
+	-moz-border-radius: 10px;
+	border-radius: 10px;
+}
+
+tr>td {
+	border: solid black;
+	background-color:/* Permalink - use to edit and share this gradient: http://colorzilla.com/gradient-editor/#f5f6f6+0,dbdce2+21,b8bac6+49,dddfe3+80,f5f6f6+100;Grey+Pipe */
+	background: rgb(245,246,246); /* Old browsers */
+	background: -moz-linear-gradient(top,  rgba(245,246,246,1) 0%, rgba(219,220,226,1) 21%, rgba(184,186,198,1) 49%, rgba(221,223,227,1) 80%, rgba(245,246,246,1) 100%); /* FF3.6-15 */
+	background: -webkit-linear-gradient(top,  rgba(245,246,246,1) 0%,rgba(219,220,226,1) 21%,rgba(184,186,198,1) 49%,rgba(221,223,227,1) 80%,rgba(245,246,246,1) 100%); /* Chrome10-25,Safari5.1-6 */
+	background: linear-gradient(to bottom,  rgba(245,246,246,1) 0%,rgba(219,220,226,1) 21%,rgba(184,186,198,1) 49%,rgba(221,223,227,1) 80%,rgba(245,246,246,1) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#f5f6f6', endColorstr='#f5f6f6',GradientType=0 ); /* IE6-9 */
+
+	-webkit-border-radius: 10px;
+	-moz-border-radius: 10px;
+	border-radius: 10px;
+
+	text-align: center;
+	font-size: 20px;
+}
+
+tr>td:hover {
+	color: white;
+}
+
+tr>td>div:hover {
+	background-color: black;
+}
+

```

Comment:
You can add style 
tr>td>div {
    width:100%;
    height:100%;
}

and add the property "padding:0" to the "tr>td" style. It will make the button hover feel more natural. You can also add the ":active" selector, to make the div look clickable : )